doc/radosgw/config-ref.rst

```diff
@@ -1115,6 +1115,44 @@ Keystone Settings
 :Type: Boolean
 :Default: ``true``
 
+
+
+LifeCycle Management Settings
```

Comment:
can we use `Lifecycle` here to be consistent with [amazon docs](http://docs.aws.amazon.com/AmazonS3/latest/dev/object-lifecycle-mgmt.html)?